Microsoft.NET.Build.Containers/Resources/Strings.resx

```diff
@@ -148,31 +148,31 @@
     <value>Token response had neither token nor access_token.</value>
   </data>
   <data name="MissingBaseImageConfigNode" xml:space="preserve">
-    <value>Expected base image to have a config node</value>
+    <value>Expected base image to have a config node.</value>
   </data>
   <data name="MissingLayerNode" xml:space="preserve">
     <value>Tried to get layer information but there is no layer node?</value>
   </data>
   <data name="MissingLinkToRegistry" xml:space="preserve">
-    <value>Need a good error for 'couldn't download a thing because no link to registry'</value>
+    <value>Need a good error for 'couldn't download a thing because no link to registry'.</value>
```

Comment:
What about

> Unable to download layer with descriptor '{digest}' from registry '{registry}' because it does not exist.

Where `digest` is the descriptor digest and `registry` is the registry display URL (aka just the domain and port if provided - we have a property for this already).

Reply:
Done :)